.github/workflows/sync-cac-oscal.yml

```diff
@@ -0,0 +1,413 @@
+name: Sync CaC content to OSCAL
+permissions:
+  contents: write
+  pull-requests: read
+on:
+  pull_request:
+    types:
+      - closed
+    branches:
+      - master
+
+jobs:
+  check-pr-message:
+    runs-on: ubuntu-latest
+    outputs:
+      run_job_check_update: ${{ steps.check_pr.outputs.run_job_check_update }}
+    steps:
+    - name: Check if the PR comes from the sync of OSCAL content
+      id: check_pr
```

Comment:
Ids are using "_" and "-". Could we keep the separator between words consistent?